docs/install/windows.rst

```diff
@@ -32,7 +32,7 @@ type:
 
 .. code:: bash
 
-   python -m venv env
+   python -m virtualenv env
```

Comment:
virtualenv must be install with pip:
```python -m pip install virtualenv
```

Reply:
Should I add a line above ```python -m virtualenv env``` saying ```python -m pip install virtualenv```? To install virtualenv

Reply:
Added!